code/go/0chain.net/smartcontract/minersc/fees.go

```diff
@@ -369,17 +369,17 @@ func (msc *MinerSmartContract) payFees(t *transaction.Transaction,
 
 	emitUpdateMiner(mn, balances, false)
 
-	if gn.RewardRoundFrequency != 0 && mb.Round%gn.RewardRoundFrequency == 0 {
-		var lfmb = balances.GetLastestFinalizedMagicBlock().MagicBlock
-		if lfmb != nil {
-			err = msc.viewChangePoolsWork(gn, lfmb, mb.Round, balances)
-			if err != nil {
-				return "", err
-			}
-		} else {
-			return "", common.NewError("pay fees", "cannot find latest magic bock")
+	//if gn.RewardRoundFrequency != 0 && mb.Round%gn.RewardRoundFrequency == 0 {
```

Comment:
We'd better not change the logic of checking if the reward round frequency, this changes will make the miners get AllMinserList from MPT every time this payFee transaction is executed in the `msc.viewChangePoolSwork`.

Reply:
Ops! that was just debugging. I wanted to force stepping through the code using the benchmark and that seemed the easiest way.